README.md

```diff
@@ -1,55 +1,16 @@
-# MultiversX SDK for JavaScript and TypeScript: wallet components
-
-Wallet components (generation, signing) for TypeScript (JavaScript).
-
-## Distribution
-
-[npm](https://www.npmjs.com/package/@multiversx/sdk-wallet)
-
-## Installation
-
-`sdk-wallet` is delivered via [npm](https://www.npmjs.com/package/@multiversx/sdk-wallet), therefore it can be installed as follows:
-
-```
-npm install @multiversx/sdk-wallet
-```
-
-## Development
-
-Feel free to skip this section if you are not a contributor.
-
-### Additional dependencies
-
-Instanbul, for code coverage:
+# This repo was moved to [sdk-core](https://www.npmjs.com/package/@multiversx/sdk-core) and will soon be deprecated
+For the update you will need to fix the imports 
 
-```
-npm install --no-save nyc
-```
-
-### Building the library
-
-In order to compile `sdk-wallet`, run the following:
-
-```
-npm install
-npm run compile
-npm run compile-browser
-```
+## For example:
+``` 
+import { UserSigner } from "@multiversx/sdk-wallet";
 
-### Running the tests
+will be 
```

Comment:
Code snippets can be improved (two, separated by the text).

> ``` 
> import { UserSigner } from "@multiversx/sdk-wallet";
> ```
> 
> should be changed to:
> 
> ```
> import { UserSigner } from "@multiversx/sdk-core";
> ```

Preview: https://github.com/multiversx/mx-sdk-js-wallet/tree/UpdateReadmeAfterUnification

:upside_down_face: